### ISSUE TITLE
[FEATURE] Menu/Message Non-square Pixel Scaling

### ISSUE THREAD

**Issue by OpenRift412** [odamex/odamex#315]
Something I've noticed is that Odamex appears to scale the menu and message graphics scale with square pixels instead of non-square pixels, like they would in vanilla Doom. This results in squished graphics that don't look right when navigating menus.

Here's some examples in comparison to PrBoom-plus.
**PrBoom-plus**
![image](https://user-images.githubusercontent.com/29780788/115978450-75104f00-a54d-11eb-8a46-080d5fc82033.png)
**Odamex**
![image](https://user-images.githubusercontent.com/29780788/115978434-5d38cb00-a54d-11eb-9089-e3a72db57726.png)

Would it be possible to have this menu scaling added to Odamex?

**Comment by leximayfield** [comment]
Indeed!  This something that sticks out to me personally like a sore thumb, and it's something I very much want to address.

I intended to look at 1.2x scaling in the `hud-rework` branch as part of a larger rework of Odamex's user interface.  Status bar graphics, HUD graphics, and indeed menu graphics would be touched by that branch.

**Comment by OpenRift412** [comment]
> Indeed! This something that sticks out to me personally like a sore thumb, and it's something I very much want to address.
> 
> I intended to look at 1.2x scaling in the `hud-rework` branch as part of a larger rework of Odamex's user interface. Status bar graphics, HUD graphics, and indeed menu graphics would be touched by that branch.

Awesome. Hope we see something of it soon.
